app/services/update_wikidata_stats.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/wikidata_summary_parser"
+
+class UpdateWikidataStats
+  def initialize(course)
+    @course = course
+    update_summaries
+    update_wikidata_stats
+  end
+
+  private
+
+  def update_summaries
```

Comment:
This method imports missing summaries rather than update any already-present ones. I suggest calling it `import_summaries`.